src/Core/Services/Implementations/I18nViewLocalizer.cs

```diff
@@ -30,8 +30,5 @@ public class I18nViewLocalizer : IViewLocalizer
         public LocalizedString GetString(string name) => _stringLocalizer[name];
         public LocalizedString GetString(string name, params object[] arguments) =>
             _stringLocalizer[name, arguments];
-
-        [Obsolete("This method is obsolete. Use `CurrentCulture` and `CurrentUICulture` instead.")]
-        public IHtmlLocalizer WithCulture(CultureInfo culture) => _htmlLocalizer.WithCulture(culture);
```

Comment:
@cscharf (@ because you're the only one that has touched this file) can you speak to what this is doing and if it is needed? Simply removing the method completely doesn't break any builds, but `WithCulture()` is gone in updated packages.